docs/infrastructure.md

```diff
@@ -0,0 +1,87 @@
+## Infrastructure
+
+### [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process)
+
+### Infrastructure Overview
+The Google Infrastructure is managed via Terraform and nearly all the kubenetes deployments and such are done via a Helm Chart. See [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process) for more in depth guide. Datarepo is deployed on Kubernetes, Google Cloud Sql (postgres 11) and contains the following containers:
```

Comment:
Suggest skipping the postgres version number here. That can evolve independently of this document.

Reply:
fixed